modules/setting/setting.go

```diff
@@ -79,32 +79,33 @@ var (
 	AppWorkPath    string
 
 	// Server settings
-	Protocol             Scheme
-	Domain               string
-	HTTPAddr             string
-	HTTPPort             string
-	LocalURL             string
-	RedirectOtherPort    bool
-	PortToRedirect       string
-	OfflineMode          bool
-	CertFile             string
-	KeyFile              string
-	StaticRootPath       string
-	StaticCacheTime      time.Duration
-	EnableGzip           bool
-	LandingPageURL       LandingPage
-	UnixSocketPermission uint32
-	EnablePprof          bool
-	PprofDataPath        string
-	EnableLetsEncrypt    bool
-	LetsEncryptTOS       bool
-	LetsEncryptDirectory string
-	LetsEncryptEmail     string
-	GracefulRestartable  bool
-	GracefulHammerTime   time.Duration
-	StartupTimeout       time.Duration
-	StaticURLPrefix      string
-	AbsoluteAssetURL     string
+	Protocol                Scheme
+	Domain                  string
+	HTTPAddr                string
+	HTTPPort                string
+	LocalURL                string
+	RedirectOtherPort       bool
+	PortToRedirect          string
+	OfflineMode             bool
+	CertFile                string
+	KeyFile                 string
+	StaticRootPath          string
+	StaticCacheTime         time.Duration
+	EnableGzip              bool
+	LandingPageURL          LandingPage
+	UnixSocketPermission    uint32
+	EnablePprof             bool
+	PprofDataPath           string
+	EnableLetsEncrypt       bool
+	LetsEncryptTOS          bool
+	LetsEncryptDirectory    string
+	LetsEncryptEmail        string
+	GracefulRestartable     bool
+	GracefulHammerTime      time.Duration
+	StartupTimeout          time.Duration
+	StaticURLPrefix         string
+	StaticURLPrefixManifest string
```

Comment:
I'm not sure, adding a special var only for manifest is the right way

Reply:
Feel free to change it! I do not know anything about go and my heart isn't connected with this pull request. All I know is that it fixes the issue for me. And I know it is not very nicely done. So: You surely don't offend me if you fix it differently

Reply:
First thanks for the pull, I think we can evolve your pull ;) together ...